test/y2network/sysconfig/config_writer_test.rb

```diff
@@ -36,7 +36,7 @@
         source:     :sysconfig
       )
     end
-    let(:old_config) { instance_double(Y2Network::Config, dns: double("dns")) }
+    let(:old_config) { instance_double(Y2Network::Config, dns: double("dns"), interfaces: nil) }
```

Comment:
I expect here more code that tests also that removal of files and other parts that is added.

BTW is only network-ng affected or also master?

Reply:
should be network-ng specific, but will check in master too.